VisualPIC/DataReading/rawDataReaders.py

```diff
@@ -193,7 +193,13 @@ def _ReadTime(self, timeStep):
 
     def _ReadUnits(self):
         # OpenPMD data always provide conversion to SI units
-        self.dataUnits = "a.u." # TODO: Get the correct unit
+        # TODO: Get the units from file
+        if self.internalName == "x" or self.internalName == "y" or self.internalName == "z":
+            self.dataUnits = "μm"
+        elif self.internalName == "ux" or self.internalName == "uy" or self.internalName == "uz":
+            self.dataUnits = "m_e*c"
+        else:
+            self.dataUnits = "a.u." 
```

Comment:
`arb.u` is usually better than `a.u.` (which also stands for atomic units, which are often seen in ionization processes in PIC codes)